guides/v2.2/config-guide/cli/logging.md

```diff
@@ -60,8 +60,48 @@ To enable database logging:
     bin/magento dev:query-log:disable
     ```
 
+1. Flush the cache.
+
+    ```bash
+    bin/magento cache:flush
+    ```
+
+## Cron logging
+
+With the release of version 2.2.8, Magento now creates a separate `cron` log. 
+Magento recently made cron logging more verbose, which provided more information but lengthened the `system.log` considerably. 
+Moving `cron` info to a dedicated log makes both logs easier to read.
+
+By default, Magento writes `cron` info to `<install_directory>/var/log/cron.log`.
+
+## Syslog logging
+
+By default, Magento writes _syslog_ logs to the operating system `syslog` file.
+
+#### To enable syslog logging:
+
+1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
```

Comment:
Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior